src/client/dfuse/dfuse.h

```diff
@@ -130,7 +130,7 @@ struct dfuse_inode_ops {
 			struct stat *attr, int to_set);
 	void (*lookup)(fuse_req_t req, struct dfuse_inode_entry *parent,
 		       const char *name);
-	void (*mkdir)(fuse_req_t req, struct dfuse_inode_entry *parent,
+	void (*mknod)(fuse_req_t req, struct dfuse_inode_entry *parent,
```

Comment:
im not familiar with fuse callbacks TBH, but why is this a replacement for mkdir?

Reply:
fuse will call either, mknod is just a more generic version of mkdir.  There are small changes in the dfuse_cont file to check if the correct file type is specified there.